ext/standaloneusers/templates/CRM/Standaloneusers/Page/Login.tpl

```diff
@@ -43,7 +44,7 @@
           window.location = response.url;
           return;
         }
-        errorMsg = response.publicError || "Unexpected error";
+        errorMsg = response.publicError || "{/literal}{ts escape="ts"}Unexpected error{/ts}{literal}";
```

Comment:
Should this be `escape="js"`? Also on line 35?

Reply:
@Sjord yes, good catch, thanks!